infrastructure/terraform/components/reporting/scripts/sql/reports/completed_comms_report.sql

```diff
@@ -1,5 +1,6 @@
 SELECT
     ri.requestitemid AS requestitemid,
+    ri.clientid AS clientid,
```

Comment:
Did we decide not to add any extra fields (e.g. requestItemRefId in the end?)

Reply:
We don't need the client ID in the select as it is in the path.

Reply:
The ticket defines the fields we want in the select.

You are missing:
request-item.requestId
request-item.requestRefId
request-item.requestItemRefId
request-item.completedDate
request-item.failedReason
request-item.sendingGroupId
request-item.sendingGroupIdVersion
request-item-plan.failedReason

and need to remove
clientid
requestitemcreateddate,

Reply:
Sorry I've not updated the fields just yet. Just kept in minimum for the purpose of testing. Will update the select with the fields as suggested.

Reply:
Updated to include additional fields